### DISCUSSION TITLE
completion does not work in zsh 😢

### DISCUSSION THREAD

**Discussion by Squirre17** [junegunn/fzf#3119] [Q&A] [answered]
```shell
# squ @ squ-virtual-machine in ~ [0:26:54] C:1
$ export FZF_COMPLETION_TRIGGER='~~'

# squ @ squ-virtual-machine in ~ [0:26:57] 
$ cd ~~ 
```
fzf version
```shell
$ fzf --version
0.20.0
```

`FZF_COMPLETION_TRIGGER` does not work when I press my \<TAB\> key...

**Comment by Ioulan** [comment, marked as answer]
Are you perhaps using ohmyzsh framework?
 
If yes, just add fzf on plugins array
`plugins=(git fzf)`

If not, find where the fzf completion file stored and manually source it
`locate fzf | grep 'zsh'`

For e.g. on fedora the completion are stored at `/usr/share/zsh/site-functions/fzf` on the other hand on Void Linux `/usr/share/fzf/completion.zsh` etc...

**Reply by Squirre17** [reply]
It works!! thanks for your help
